### PR TITLE
feat: mapping entities relationships.

### PR DESCRIPTION
**Pull request by RenatoBorgess**
This group of entities are necessary to create the starting end-points for the insurance company module.

### CODE REVIEW

File: src/main/java/com/cms/claimmanagement/surveyor/repository/Surveyor.java [hunk 1]
```diff
@@ -1,18 +1,63 @@
...
+    @Column(name = "surveyor_id")
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
-
-    private Long surveyorId;
-
+    private Long id;
+    @OneToMany(mappedBy = "surveyor")
+    private List<ClaimDetails> claimDetails;
     String firstName;
```
Comment (scaiowalker) [inline src/main/java/com/cms/claimmanagement/surveyor/repository/Surveyor.java]: why this attribute is not private?

File: src/main/java/com/cms/claimmanagement/insuranceCompany/repository/PolicyRequestData.java [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
-import jakarta.validation.constraints.Email;
 import jakarta.validation.constraints.NotNull;
-import jakarta.validation.constraints.Pattern;
 
 import java.time.LocalDate;
 
-public record PolicyData(
+public record PolicyRequestData(
 
         String InsuredFirstName,
```
Comment (scaiowalker) [inline src/main/java/com/cms/claimmanagement/insuranceCompany/repository/PolicyRequestData.java]: following the java convension, classes attributes should start with lower case.

File: src/main/java/com/cms/claimmanagement/insuranceCompany/repository/ClaimDetails.java [hunk 3]
```diff
@@ -12,18 +13,103 @@
 
 @Entity
 public class ClaimDetails {
+
     @Id
     @Size(min = 10, max = 10)
-    private Long claimId;
-    //many-to-one
-    private String policyNo;
+    private String claimId;
```
Comment (scaiowalker) [inline src/main/java/com/cms/claimmanagement/insuranceCompany/repository/ClaimDetails.java:19]: I know it doesnt change anything, but to increase readability we usually keep all of the attributes together and after the last one we create getters and setter.

**Review by scaiowalker — CHANGES_REQUESTED** [review]
Great job! The comments I've added are to keep the project tidy. :)

**Comment by RenatoBorgess** [comment]
Keep the comments coming, they are very helpful!

**Review by scaiowalker — APPROVED** [review]

**Pull request merged**